env/rc.sh

```diff
@@ -1,2 +1,2 @@
 export CBIOPORTAL_URL="http://www.cbioportal.org/beta"
-export GENOME_NEXUS_URL="http://genomenexus.herokuapp.com"
+export GENOME_NEXUS_URL="http://genomenexus.org/beta"
```

Comment:
Should this be the default URL for RC environment?

Reply:
yeah that makes sense